_data/glossary/kk.yaml

```diff
@@ -0,0 +1,8 @@
+id: kk
+name: k/k
+tags:
+- community
+short-description: >
+  k/k is short for the kubernetes/kubernetes repo
+long-description: >
+  k/k is frequently used in communications to the community (comments, SIG meetings, etc.). It is a shortened moniker for the kubernetes/kubernetes core code repository.
```

Comment:
Perhaps add a link to the [kubernetes/kubernetes](https://github.com/kubernetes/kubernetes) repo.